src/app/hooks/useAddCard.ts

```diff
@@ -24,6 +25,12 @@ export default function useCodeCard(): {
       )
     );
   }
+  function editCodeCard(oldCodeCard: CodeCards, newCodeCard: CodeCards) {
+    setCodeCards([
+      ...codeCards.filter((codeCard) => codeCard.title !== oldCodeCard.title),
```

Comment:
I would've preferred this with another variable assignment in between, just for readability